arcgis-ios-sdk-samples/Search/Find place/FindPlaceViewController.swift

```diff
@@ -199,20 +192,20 @@ class FindPlaceViewController: UIViewController {
     }
     
     private func fetchSuggestions(_ string: String, suggestionType: SuggestionType, textField: UITextField) {
-        // cancel previous requests
+        // Cancel previous requests.
         if self.suggestRequestOperation != nil {
```

Comment:
There were some discussion on how to handle `AGSCancelable` here: https://github.com/Esri/arcgis-runtime-samples-ios/pull/1074#discussion_r641043335

But I tend not to make these sort of changes in this PR.

Reply:
Yes, if we choose to update the `AGSCancelable` issues, they will be in a separate PR.